lib/inspec/resources/linux_audit_system.rb

```diff
@@ -0,0 +1,81 @@
+require "inspec/resources/command"
+module Inspec::Resources
+  class LinuxAuditSystem < Inspec.resource(1)
+    # Resource's internal name.
+    name "linux_audit_system"
+
+    # Restrict to only run on the below platforms (if none were given,
+    # all OS's and cloud API's supported)
+    supports platform: "linux"
+
+    desc "Use the `linux_audit_system` Chef InSpec audit resource to test the configuration of linux audit system."
+
+    example <<~EXAMPLE
+      describe linux_audit_system do
+        it { should be_enabled }
+        it { should be_running }
+        its("rules") { should include "-w /etc -p wa" }
+        its("rules") { should include %r{-w /etc -p wa} }
+        its("rules") { should include %r!-w /etc -p wa! }
+      end
+    EXAMPLE
+
+    attr_reader :auditctl_utility
+
+    # Resource initialization.
+    def initialize
+      skip_resource "The `linux_audit_system` resource is not yet available on your OS." unless inspec.os.linux?
+      @auditctl_utility = find_auditctl_or_error
+    end
+
+    # Resource appearance in test reports.
+    def to_s
+      "linux_audit_system"
+    end
+
+    # The be_enabled matcher checks if the auditing is enabled.
+    # The enabled flag 1 indicates that the auditing is enabled.
+    def enabled?
+      auditctl_cmd = inspec.command("#{auditctl_utility} -s | grep enabled")
```

Comment:
Please let me know if we would like to change the way we fetch the report status information for checking enabled and pid values.